src/EnrollDuoController.js

```diff
@@ -38,9 +38,11 @@ function (Okta, Duo, Q, FormController, Footer) {
 
       getInitOptions: function () {
         return this.doTransaction(function (transaction) {
-          return transaction
-          .getFactorByTypeAndProvider('web', 'DUO')
-          .enrollFactor();
+          var factor = _.findWhere(transaction.factors, {
+            factorType: 'web',
+            provider: 'DUO'
+          });
+          return factor.enroll();
```

Comment:
nit: I think we should make a function out of this and put it in FactorUtil.js (or some util), since it is being used everywhere.